packages/lodestar/src/sync/utils/sync.ts

```diff
@@ -65,6 +67,20 @@ export function isValidFinalizedCheckPoint(peers: IReputation[], finalizedCheckp
   return peerCount >= (validPeers.length / 2);
 }
 
+export function isValidPeerForInitSync(config: IBeaconConfig, myState: BeaconState, peerStatus: Status): boolean {
+  if (!peerStatus) {
+    return false;
+  }
+  // TODO: compare fork_digest in the latest spec?
+  if (!peerStatus.finalizedRoot || peerStatus.finalizedEpoch <= GENESIS_EPOCH) {
```

Comment:
I just realized this check is also not ok because if we are trying to sync two peers locally and peer 1 is at block 5(no finalized epochs), sync won't start

Reply:
oh I thought that's what we expect as in the issue description: `If we are syncing from node/s that aren't yet finalized, we should skip initial sync`

Reply:
I think, I've pushed some fix to regular sync branch to handle this inside initialSync. Basically it would enter initial sync, figure out there is no peer with finalized epoch and mark initial sync compleated. We should probably close that issue with "regular sync" PR merge. This PR is still valid with just last check.

There is a lot of cases we should handle that, like you skipped initial sync but you discovered peer with higher finalized epoch, you should try to initial sync again